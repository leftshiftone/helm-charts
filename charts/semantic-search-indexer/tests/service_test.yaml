suite: test service
templates:
  - service.yaml
tests:
  - it: default
    asserts:
      - isKind:
          of: Service
      - equal:
          path: metadata.name
          value: RELEASE-NAME-semantic-search-indexer-app
      - matchSnapshot:
          path: metadata.labels
      - matchSnapshot:
          path: metadata.annotations
      - matchSnapshot:
          path: spec
