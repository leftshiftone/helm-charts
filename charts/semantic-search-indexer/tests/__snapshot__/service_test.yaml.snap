default:
  1: |
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: semantic-search-indexer
    app.kubernetes.io/version: 0.0.0
    application: semantic-search-indexer
    helm.sh/chart: semantic-search-indexer-0.0.0
  2: |
    null
  3: |
    ports:
    - name: http
      port: 80
      protocol: TCP
      targetPort: http
    selector:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/name: semantic-search-indexer
    type: ClusterIP
